rootdir/etc/init.halium.rc

```diff
@@ -44,7 +44,7 @@ on property:sys.boot_completed=1
 on charger
     start charger
 
-service charger /sbin/charger
+service charger /bin/charger
```

Comment:
/bin is a symlink, so I think for consistency it would be better to put full path (/system/bin/charger) instead.

Reply:
Hmm....strange in volla x we not have simlink...